test.js

```diff
@@ -0,0 +1,107 @@
+let expect = require("chai").expect;
```

Comment:
It would be nice to run these tests in node as well as browser(in Karma) since browser is the intended target for this (Not needed for now. Can add it when we get into the Angular playground repo).